File: hafta11/uyg1.java
public class uyg1 {

    // Altprogram başlığı (header): public static int carp(int x, int y)
    public static int carp(int x, int y) {  // resmi parametreler: x, y
        return x * y;
    }

    public static void main(String[] args) {
        int a = 4;
        int b = 7;

        // Gerçek parametreler: a, b
        int sonuc = carp(a, b);
        System.out.println("Çarpım: " + sonuc);
    }
}

File: hafta11/uyg1.c
#include <stdio.h>

// Altprogram başlığı (header): int toplama(int a, int b)
int toplama(int a, int b) {  // resmi parametreler: a, b
    return a + b;
}

int main() {
    int sayi1 = 5;
    int sayi2 = 3;
    int sonuc;

    // Gerçek parametreler: sayi1, sayi2
    sonuc = toplama(sayi1, sayi2);
    printf("Toplam: %d\n", sonuc);
    return 0;
}
